tokio-console/src/state/tasks.rs

```diff
@@ -217,14 +217,16 @@ impl TasksState {
                     warnings: Vec::new(),
                     location,
                 };
-                task.lint(linters);
                 Some((id, task))
             });
 
         for (stats, mut task) in self.tasks.updated(stats_update) {
             tracing::trace!(?task, ?stats, "processing stats update for");
             task.stats = stats.into();
-            task.lint(linters);
+        }
+
+        for (_, task) in self.tasks.iter() {
```

Comment:
I think that this strategy probably isn't ideal. I'm concerned that we will be adding too much additional processing on every update.

Especially as more lints are added, a situation in which there are lots and lots of tasks. In the case that most of these don't change in every update (which is probably a common situation), this will introduce a large amount of unnecessary overhead. Especially given that most of the lints won't change result unless there is a change in the task itself.